DECISION-RESEARCH.md

```diff
@@ -18,7 +18,20 @@
 
 # Trailblaze (September 2020)
 
-TBA
+## Title / Revision info in response bodies
+
+**Decision:Leave this meta infomation out of the response bodies for the first iteration.**
+
+- `lastrevid` & `modified`
```

Comment:
`modified` is already included as part of the `Entity` interface in the specs' code base. Is that a mistake? should we change that?

Reply:
Yes we should remove that!